src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockWorkspaceServer.scala

```diff
@@ -364,6 +368,20 @@ object MockWorkspaceServer {
           .withBody(mockMethodConfigs.toJson.prettyPrint)
           .withStatusCode(OK.intValue))
 
+    MockWorkspaceServer.workspaceServer.
+      when(
+        request()
+          .withMethod("PATCH")
+          .withPath(s"/workspaces/%s/%s"
+          .format(mockValidWorkspace.namespace.get, mockValidWorkspace.name.get))
+          .withBody(mockUpdateAttributeOperation.toJson.prettyPrint)
+          .withHeader(authHeader))
+      .respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(OK.intValue)
+      )
+
```

Comment:
It isn't too bad creating a negative test case. If you define a when case that listens for the values `mockInvalidWorkspace`, and then responds with what Rawls would respond with, you could then call that invalid case in your test class. 

If you did do that, you'd need to modify this when handler to only respond to a valid case, such as the values in `mockValidWorkspace` and call it appropriately in the test class.